### PR TITLE
add unit test for file dataStream

### LINKED ISSUE #78: Add tests for local file import.

### PR DESCRIPTION
**Pull request by screeley44**
fixes #78 

Initial WIP to adda unit test for file/local dataStream

### CODE REVIEW

**Comment by copejon** [comment]
Typo in the travis script?

$ sudo minikube start --vm-driver=none --kubernetes-version=**v**$K8S_VER

**Comment by jeffvance** [comment]
This looks good. Just remove the _ "file does not exist"_ test case.

**Comment by screeley44** [comment]
@jeffvance ack - removed and squashed

File: pkg/importer/importer_test.go [hunk 1]
```diff
@@ -88,6 +94,14 @@ var _ = Describe("Importer", func() {
 		})
 
 		tests := []testT{
+			{
+				descr:       "use file",
+				filename:    "test-local",
+				createFile:  false,
+				expected:    "file dataStream",
+				endpoint:    "file:///tmp/importer-test/",
```
Comment (jeffvance) [inline pkg/importer/importer_test.go]: why use a directory path here vs. a file? I know it's mock but maybe something like `file:///tmp/fake-file` is clearer? I got confused by _/tmp/importer-test/_, which is a `const`, is the destination path for the copy.

Comment (screeley44) [reply pkg/importer/importer_test.go]: yep, agreed, just an oversight, changed in latest commit

**Pull request merged**